playbooks/openshift/delete-cluster.yml

```diff
@@ -1,4 +1,7 @@
 ---
+- hosts: seed-hosts
+  roles:
+  - openshift-pv-cleanup
```

Comment:
This needs to be referenced from `openshift-ansible-contrib` - i.e.: add the `roles/openshift-ansible-contrib` to the path